app/routes/__index/domains/new.tsx

```diff
@@ -1,5 +1,5 @@
 import { Container, Heading, Text } from '@chakra-ui/react';
-import { RecordType } from '@prisma/client';
+import { RecordStatus, RecordType } from '@prisma/client';
```

Comment:
It seems like `RecordStatus` is never being used. I would remove it, so ESLint doesn't complain.